lib/manager/travis/readme.md

```diff
@@ -1 +1,28 @@
-This manager is intended to keep Travis config files (`.travis.yml`) up-to-date. Currently it manages only the `node_js` section of files only.
+This manager is intended to keep Travis config files (`.travis.yml`) up-to-date.
+Currently it manages only the `node_js` section of files only.
```

Comment:
```suggestion
This manager is intended to keep Travis config files (`.travis.yml`) up-to-date, this file controls the CI build environment.
Currently Renovate can only update the `node_js` section of this file.
```

- Add some explanation about what the `.travis.yml` file controls.
- Reword wonky sentence.